basis/colors/constants/constants.factor

```diff
@@ -19,7 +19,8 @@ IN: colors.constants
 MEMO: colors ( -- assoc )
     "resource:basis/colors/constants/rgb.txt"
     "resource:basis/colors/constants/factor-colors.txt"
-    [ utf8 file-lines parse-colors ] bi@ assoc-union ;
+    "resource:basis/colors/constants/solarized-colors.txt"
```

Comment:
It's a very good colour theme. I think it deserves to be among the official constants. The constants for the light theme are already here, and the other constants seem to have just be taken from the X windows system.